content/github/visualizing-repository-data-with-graphs/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Visualizing repository data with graphs
+intro: You can visualize data effectively using graphs.
```

Comment:
Hi @Perkles - thanks for adding an introduction to this category.

When we add an introduction, we try to use different terminology from the article title. I can see you've done this in earlier pull requests, but maybe this one was more difficult.

Could you update this introduction to use other terms? For example, maybe some of these might work: display, view, information, charts.

Reply:
Hello @felicitymay 👋 
Sure! I've rephrased the intro almost entirely. 
Let me know what you think about it.